config/cross_section_config.py

```diff
@@ -552,7 +552,7 @@ def __fill_defaults_13TeV__( self ):
         path_to_files = self.path_to_files
 
         # self.new_luminosity = 40.028  # pb^-1
-        self.new_luminosity = 41.62912  # pb^-1
+        self.new_luminosity = 71.515  # pb^-1
```

Comment:
Did not see anything new, but on September the 17th a new version was released. I will assume that it gives the correct number for now.